dm/worker/server.go

```diff
@@ -177,6 +184,42 @@ func (s *Server) Start() error {
 	return terror.ErrWorkerStartService.Delegate(err)
 }
 
+func (s *Server) syncMasterEndpoints(ctx context.Context) {
+	lastClientUrls := []string{}
+	clientURLs := []string{}
+
+	updateF := func() {
+		clientURLs = clientURLs[:0]
+		resp, err := s.etcdClient.MemberList(ctx)
+		if err != nil {
+			log.L().Error("can't get etcd member list", zap.Error(err))
+			return
+		}
+
+		for _, m := range resp.Members {
+			clientURLs = append(clientURLs, m.GetClientURLs()...)
+		}
+		if utils.NonRepeatStringsEqual(clientURLs, lastClientUrls) {
+			log.L().Info("etcd member list doesn't change", zap.Strings("client URLs", clientURLs))
```

Comment:
```suggestion
			log.L().Debug("etcd member list doesn't change", zap.Strings("client URLs", clientURLs))
```

this can still be `Debug`?